app/controllers/api/functions.php

```diff
@@ -1311,11 +1715,12 @@
     ->label('sdk.response.model', Response::MODEL_VARIABLE)
     ->param('functionId', '', new UID(), 'Function unique ID.', false)
     ->param('key', null, new Text(Database::LENGTH_KEY), 'Variable key. Max length: ' . Database::LENGTH_KEY  . ' chars.', false)
-    ->param('value', null, new Text(8192, 0), 'Variable value. Max length: 8192 chars.', false)
+    ->param('value', null, new Text(8192), 'Variable value. Max length: 8192 chars.', false)
```

Comment:
Make sure to add the 0 min back:

```suggestion
    ->param('value', null, new Text(8192, 0), 'Variable value. Max length: 8192 chars.', false)
```